Units/parser-lex.r/simple-lex.d/input.l

```diff
@@ -123,7 +123,7 @@
 #include <stdio.h>
 #include <string.h>
 
-#ifdef WIN32
+#ifdef _WIN32
```

Comment:
This is a part of test input. Please don't touch this.

Reply:
Would not the test fail if it is kept unchanged?

Reply:
No. But this is a part of test input. Please don't touch this.
This change makes your original intent of this pull request unclear.

Reply:
I have removed that change from the commit.